nipoppy_cli/docs/source/user_guide/processing.md

```diff
@@ -0,0 +1,132 @@
+# Running processing pipelines
+
+Just like with the BIDS conversion pipelines, Nipoppy uses the {term}`Boutiques framework <Boutiques>` to run image processing pipelines. By default, new Nipoppy datasets (as created with [`nipoppy init`](<project:../cli_reference/init.md>)) are populated with descriptor files and default invocation files for the following processing pipelines:
+- [fMRIPrep](https://fmriprep.org/en/stable/), a pipeline for preprocessing anatomical and functional MRI data.
+- [MRIQC](https://mriqc.readthedocs.io/en/latest/index.html), a pipeline for automated quality control (QC) metric extraction
+
+```{note}
+Although fMRIPrep and MRIQC are both [BIDS Apps](https://bids-apps.neuroimaging.io/about/), Nipoppy can also be used to run pipelines that are not BIDS Apps. Custom pipelines can be added by creating a Boutiques descriptor file and modifying the global configuration file accordingly.
+```
+
+## Summary
+
+### Prerequisites
+
+- A Nipoppy dataset with a valid global configuration file and an accurate manifest
+    - See the [Quickstart guide](../quickstart.md) for instructions on how to set up a new dataset
+- Raw imaging data organized according to the {term}`BIDS` standard in the {{dpath_bids}} directory
+    - See <project:bids_conversion.md>
+
+```{include} ./inserts/apptainer_stub.md
+```
+
+### Data directories
+
+| Directory | Content description |
+|---|---|
+| {{dpath_bids}} | **Input** -- {{content_dpath_bids}} |
+| {{dpath_derivatives}} | **Output** -- {{content_dpath_derivatives}} |
+
+Within the {{dpath_derivatives}} directory, output files for a specific pipeline are organized like this:
+```{literalinclude} ./inserts/pipeline_derivatives.txt
+---
+class: no-copybutton
+---
+```
+
+### Commands
+
+- Command-line interface: [`nipoppy run`](<project:../cli_reference/run.md>)
+- Python API: {class}`nipoppy.workflows.PipelineRunner`
+
+### Workflow
+
+1. Nipoppy will loop over all participants/sessions that *have* BIDS data according to the {term}`doughnut file` but *have not* yet successfully completed the pipeline according the the imaging derivatives bagel file
+    - An existing, out-of-date doughnut file can be updated with [`nipoppy doughnut --regenerate`](../cli_reference/doughnut.md)
```

Comment:
Why / when would `doughnut` be out of date?

Reply:
For example if BIDS conversion ran without error (thus updating entries in the doughnut) but didn't produce a valid BIDS dataset. Then the user needs to delete the `bids` directory and rerun the conversion but before that the doughnut needs to be updated so that the `in_bids` column becomes `False` again